pkg/cloud/cloudpb/external_storage.proto

```diff
@@ -60,6 +64,9 @@ message ExternalStorage {
     // chain. These roles will be assumed in the order they appear in the list
     // so that the role specified by RoleARN can be assumed.
     repeated string delegate_role_arns = 13 [(gogoproto.customname) = "DelegateRoleARNs"];
+
```

Comment:
Can we add comments to these fields, its okay if they're even the same as the above two fields. Also should we add a TODO to remove the above two fields in 23.2? I assume we're migrating away from them entirely in favour of the new fields.

Reply:
Done, added the comments and TODOs.